content/en/flux/use-cases/gh-actions-auto-pr.md

```diff
@@ -32,7 +32,7 @@ To create the pull request whenever automation creates a new branch, in your man
 # ./.github/workflows/staging-auto-pr.yaml
 name: Staging Auto-PR
 on:
-  create:
+  push:
```

Comment:
This would run this workflow whenever commits are pushed to this branch which is also not desirable. I think the only way to accomplish this is to use the `create` event and add appropriate `if` conditions to the jobs to check the ref name.